server/src/main/java/org/exoplatform/chat/server/ChatServer.java

```
@@ -457,6 +457,22 @@ else if (targetUser.startsWith(ChatService.TEAM_PREFIX))
 
   @Resource
   @MimeType("text/plain")
+  @Route("/deleteTeamRoom")
+  public Response.Content deleteTeamRoom(String user, String token, String room, String dbName) throws IOException {
+    if (!tokenService.hasUserWithToken(user, token, dbName)) {
+      return Response.notFound("Petit malin !");
+    }
+    try {
+      chatService.deleteTeamRoom(room, user, dbName);
+    } catch (Exception e) {
+      LOG.severe("Impossible to delete Team Room [" + room + "] : " + e.getLocalizedMessage());
```

We generally do not use getLocalizedMessage but getMessage, to avoid having chinese messages in logs :)

Also, you should not swallow the exception : LOG.severe("...", e)